frontend/src/pages/DashboardPage/Slider/Slider.jsx

```diff
@@ -8,13 +9,27 @@ const Slider = ({ graphData, maxValue }) => {
     const handleSliderChange = (event) => {
         setSliderValue(parseFloat(event.target.value));
         console.log('Slider Value:', event.target.value); // For debugging
+        updateSliderStyle(event.target, event.target.value);
+    };
+
+    const updateSliderStyle = (slider, value) => {
```

Comment:
Can you explain this part of code

Reply:
Yeah so it takes the slider value from above, and sends it to the css in a way (it directly changes the styling in the jsx file) so that when value is less than 0.33, its green. between 0.33 and 0.66 is yellow and 0.66 to 1 is red